code/crates/test/tests/it/value_sync.rs

```diff
@@ -343,6 +343,47 @@ pub async fn start_late_rotate_epoch_validator_set() {
         .await
 }
 
+#[tokio::test]
+pub async fn sync_only_fullnode_without_consensus() {
+    const HEIGHT: u64 = 8;
+
+    let mut test = TestBuilder::<()>::new();
+
+    // First two nodes are normal validators that will drive consensus
+    test.add_node()
+        .with_voting_power(10)
+        .start()
+        .wait_until(HEIGHT)
+        .success();
+
+    test.add_node()
+        .with_voting_power(10)
+        .start()
+        .wait_until(HEIGHT)
+        .success();
+
+    // Third node is a sync-only full node (0 voting power, consensus disabled)
+    // It should be able to sync values but not participate in consensus
+    test.add_node()
+        .with_voting_power(0) // Non-validator
```

Comment:
We can use the `full_node` method to make this more explicit:

```suggestion
        .full_node()
```